src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -43,13 +43,49 @@ def _get_invalid_csv_error_msg(line_snippet, file_name):
     return INVALID_CONTENT_FORMAT_ERROR.format(line_snippet=line_snippet, file_name=file_name, content_type='CSV')
 
 
+def _get_csv_content_type(content_type_cfg_val):
+    """
+    Returns CSV if content_type_cfg_val is
```

Comment:
Minor: PEP8, make this a command rather than desc ie ```Return CSV if content_type ...```

See examples here: https://www.python.org/dev/peps/pep-0257/